README.md

```diff
@@ -0,0 +1,24 @@
+# Data Plane Gateway
+
+[Gazette](https://github.com/gazette/core) exposes several GRPC services we'd like to provide broader access to.
+1. We'd like to expose REST versions of these services.
+2. We'd like to enforce authentication and authorization. Coming soon:tm:.
+
+![Data Plane Gateway](docs/data-plane-gateway-v1.png)
+
+## Rest Gateway
+
+We use the [grpc-gateway](https://github.com/grpc-ecosystem/grpc-gateway/) project to generate rest handlers from the protobuf definitions found in Gazette. We also generate swagger documentation which we then use to generate a typescript client.
```

Comment:
> We also generate swagger documentation which we then use to generate a typescript client.

I believe the _s_ in _swagger_ and the first _t_ and _s_ in _typescript_ should be capitalized. The latter suggestion applies to line 18 as well. Calling this out for general awareness and record as the change is non-essential.